docs/handbook/image-file-formats.rst

```diff
@@ -1497,6 +1497,11 @@ The :py:meth:`~PIL.Image.Image.save` method can take the following keyword argum
     image, will determine the physical dimensions of the page that will be
     saved in the PDF.
 
+**dpi**
+    A tuple of (x_resolution, y_resolution), with inches as the resolution
```

Comment:
```suggestion
    A tuple of ``(x_resolution, y_resolution)``, with inches as the resolution
```

Reply:
This is actually copying existing documentation. i've created PR #6969 to update the original.